contracts/Comet.sol

```diff
@@ -1323,4 +1323,4 @@ contract Comet is CometMainInterface {
             default { return(0, returndatasize()) }
         }
     }
-}
+}
```

Comment:
This is the change to `Comet.sol`?

Reply:
I think there was a vestigial `import hardhat/console.log;` in this file at one point

Reply:
Should we just get rid of the diff on this one?

Reply:
sure; added to #480